content/2015.12.what-can-kinto-do.rst

```diff
@@ -23,14 +23,124 @@ Here is an attempt to fix this.
+Said differentely, you store your data locally for your application and **Kinto
+will make this data available to different devices**.
+
+That means that you can use Kinto with React, Angular, JQuery (in the
+JavaScript world) or any framework (or no framework at all) in any language.
+**Kinto abstracts away the synchronisation of the application's data**.
+
+Hmm, and what about my use-case?
+################################
+
+The idea here is mainly to show you what is already possible with very little
+efforts using the tooling we have put together. You can leverage the
+`Kinto.js <https://kintojs.readthedocs.org>`_ client, as it allows you to work
+with Kinto really easily, but ultimately you don't need it. It's just here to
+make it really easy for you to integrate with the HTTP APIs exposed by Kinto.
+
+Synchronize your data
+=====================
+
+If there is one thing to take way with you, that would be this: if you need to 
+**synchronise** your data between different devices, there are chances that
+Kinto is a good fit.
+
+That's useful if you want to synchronize data between devices, and also between
+users. You have the data on one location, and you want it in a different one.
+
+With Kinto.js, this is as simple as calling `.sync()` on the Kinto instance.
+
+.. code-block:: js
+
+   const tasks = new Kinto({
+       remote: "https://kinto.dev.mozaws.net/v1"
+   }).collection("tasks");
+
+   XXX
+
+
+Storage
+=======
+
+You're not only syncing your data between different locations, you're actually
+storing it somewhere, on a location you chosed.
+
+You can access this data later on when you need. The cool thing here is
+that you can access this data using an HTTP API that speaks JSON.
+
+We tried very hard to make the protocol as easy as possible, and are pretty
+happy with our current design, but if you have ideas on how to make it evolve,
+`we would be really happy to hear them
+<https://kiwiirc.com/client/irc.freenode.net/?#kinto>`_!
+
+Here's an example of how to retrieve your data using the `HTTPie
+<https://github.com/jkbrzt/httpie>`_ client.
+
+.. code-block:: bash
+
+    $ http get https://kinto.dev.mozaws.net/v1/buckets/default/collections/kinto_demo_calendar/records --auth="null:notsecret"
+
+    {
+        "data": [
+            {
+                "end": "2016-01-07T09:30:00", 
+                "id": "1f5d1390-2647-442d-8bc4-5ecc6a4f8beb", 
+                "last_modified": 1451997254315, 
+                "start": "2016-01-07T09:00:00", 
+                "title": "Visite maison"
+            }, 
+            {
+                "end": "2016-01-06", 
+                "id": "0dadb484-259b-482d-8cb8-0e94d743d1c6", 
+                "last_modified": 1451997234471, 
+                "start": "2016-01-05", 
+                "title": "RDV Osthéo"
+            }
+        ]
+    }
+
+
+Permissions
+===========
+
+Sometimes, you want your data to remain yours, and some other times you want to
```

Comment:
@phrawzty I don't know if I am right but we shouldn't have an Oxford coma here right?